v0/sources/message.js

```diff
@@ -31,6 +30,9 @@ class Message {
   }
 
   setProperty(name, value) {
+    if (!this.properties) {
+      this.properties = {};
```

Comment:
why do we need this?

Reply:
Not needed. May be late-night coding added this. 🙏 Fixed this.